Tests/Marketplace/search_and_install_packs.py

```diff
@@ -441,12 +444,13 @@ def install_all_content_packs_for_nightly(client: demisto_client, host: str, ser
     install_packs(client, host, all_packs, is_nightly=True)
 
 
-def install_all_content_packs(client: demisto_client, host: str):
+def install_all_content_packs(client: demisto_client, host: str, server_version=None):
```

Comment:
```suggestion
def install_all_content_packs(client: demisto_client, host: str, server_version):
```
As the Upload-Flow is the only one using this function, let's keep this arg as mandatory

If you think its not mandatory, then a verification that isn't empty should be done before line 467